### PR TITLE
Visual tweaks

### PR DESCRIPTION
**Pull request by AbraaoMota**
Several visual enhancements to the page

### CODE REVIEW

File: frontend/webclient/index.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <html>
 
   <head>
-  
+
+    <title>ALABNO</title>
```
Comment (giuliojiang) [inline frontend/webclient/index.html:5]: finally ;)

File: frontend/webclient/script/app.js [hunk 2]
```diff
@@ -15,7 +15,9 @@ theapp.config(['$routeProvider', function($routeProvider) {
     }).
 
     otherwise({
-      redirectTo: '/professor'
+      // redirectTo: '/professor'
+      templateUrl: 'landing.htm'
```
Comment (giuliojiang) [inline frontend/webclient/script/app.js:18]: ok. uncomment?

File: frontend/webclient/script/handlers.js [hunk 3]
```diff
@@ -110,26 +110,31 @@ $handlers.handle_job_list = function(msgobj) {
...
+      color = "blue";
+      glyphicon = "time";
     } else if (in_job_status == "error") {
       color = "red";
+      glyphicon = "warning-sign";
     } else if (in_job_status == "processing") {
-      color = "yellow";
+      color = "orange";
+      glyphicon = "cog";
     }
```
Comment (giuliojiang) [inline frontend/webclient/script/handlers.js:124]: great

File: frontend/webclient/script/handlers.js [hunk 4]
```diff
@@ -285,8 +290,27 @@ $handlers.handle_commits = function(msgobj) {
...
+    if (color === 'black') {
+      msgobj.alert_type = 'info';
+    } else if (color === 'green') {
+      msgobj.alert_type ='success';
+    } else if (color === 'yellow') {
+      msgobj.alert_type = 'warning';
+    } else if (color === 'red') {
+      msgobj.alert_type = 'danger';
+    }
+
```
Comment (giuliojiang) [inline frontend/webclient/script/handlers.js:305]: ok, server not changed

File: frontend/webclient/script/handlers.js [hunk 5]
```diff
@@ -285,8 +290,27 @@ $handlers.handle_commits = function(msgobj) {
...
     var checker = msgobj;
+
     $globals.top_scope.statusinformation.unshift(msgobj);
+    console.log("STATUS ARR BEFORE: " + $globals.top_scope.statusinformation);
+    // only keep latest 3 messages
+    $globals.top_scope.statusinformation = $globals.top_scope.statusinformation.slice(0,3);
```
Comment (giuliojiang) [inline frontend/webclient/script/handlers.js:310]: ok, It seems to work for me, maybe remove the console logs?

**Review by giuliojiang — APPROVED** [review]

File: frontend/webclient/script/handlers.js (hunk 4 above)
Comment (AbraaoMota) [reply frontend/webclient/script/handlers.js:305]: to double check, do you mean this is fine?

**Review by helicopter88 — APPROVED** [review]
LGTM

**Pull request merged**